service/debugger/debugger.go

```diff
@@ -557,14 +557,59 @@ func (d *Debugger) ClearBreakpoint(requestedBp *api.Breakpoint) (*api.Breakpoint
 	d.processMutex.Lock()
 	defer d.processMutex.Unlock()
 
-	var clearedBp *api.Breakpoint
-	bp, err := d.target.ClearBreakpoint(requestedBp.Addr)
+	var bps []*proc.Breakpoint
+	var remaddrs []string
+	var errs []error
+	addrnum := 0
```

Comment:
Do we need this? Can't we infer from `len(bps) == 0` that we weren't able to clear any of the breakpoints?